main.py

```diff
@@ -1,12 +1,199 @@
+        "3": "Create a new customer profile",
+        "4": "Edit a customer profile", 
+        "5": "Delete a customer", 
+        "6": "See all videos",
+        "7": "See all the information about a video",
+        "8": "Create a new video to add to the store",
+        "9": "Edit a video's information",
+        "10": "Delete a video",
+        "11": "Check-out a video",
+        "12": "Check-in a video",
+        "13": "List all options",
+        "14": "Quit"
+        }
+
+    print_hashtags()
+    print("Welcome to the Retro Video Store!!!")
+    print("Here are your options......")
+    print_hashtags()
+    
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    print_hashtags()
+
+    return options
+
+
+def make_choice(options, retro_video_store):
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("What would you like to do? Select 13 to see all options again")
+        choice = input("Make your selection using the option number: ")
+    
+    return choice
+
+def run_cli(play=True):
+
+    #initialize retro_video_store
+    retro_video_store = Video_Store(url="http://127.0.0.1:5000")
+    
+    # print choices
+    options = list_options()
+
+    while play==True:
+
+        # get input and validate
+        choice = make_choice(options, retro_video_store)
+
+        retro_video_store.print_selected()
+
+        if choice=='1':
+            print_hashtags()
+            print("Here are all of the customers we have in our database.")
+            for customer in retro_video_store.list_of_customers():
+                print(customer)
+
+        elif choice=='2':
+            id=input("What is the id of the customer you want information about? ")
+            if id.isnumeric():
+                id=int(id)
+                print("Here is this customers information:", retro_video_store.get_customer(id=id))
+            else:
+                print("Please enter the number id.")
+
+        elif choice=="3":
+            name=input("What is the name of this new customer? ")
+            postal_code=input("What is their postal code? ")
+            phone=input("What's their phone number that they'll answer the most? ")
+            response = retro_video_store.create_customer(name=name, postal_code=postal_code, phone=phone)
+            print("New Customer id:", response["id"])
+
+        elif choice=='4':
+            print(f"Time to update someone's information!: {retro_video_store.selected_customer}")
+            name=input("What is their new name? ")
+            postal_code=input("What is their new postal code? ")
+            phone=input("What is their new phone number (if they changed it)? ")
+            response = retro_video_store.update_customer(id, name=name, postal_code=postal_code, phone=phone)
+
+            print_hashtags()
+            print("Updated customer name:", response["name"])
```

Comment:
I really like how I could enter an empty postal code here. You always want to give users the option of updating a single attribute rather than all.